src/lib/TranscriptEditor/TimedTextEditor/export-adapters/txt/index.js

```diff
@@ -0,0 +1,4 @@
+export default (blockData) => {
+  const lines = blockData.blocks.map(x => x.text);
+  return lines.join(' ');
```

Comment:
I'd add a line breaks to respect the paragraphs, if the user edits the text, and arranges it with the desired paragraph breaks, it be good to preserve this when exporting in plain text.

```suggestion
  return lines.join('\n\n');
```

When we add speaker names at paragraph level we can revisit this, and add more option to the plain text output eg optional speakers, and timecodes and paragraph level.